src/go/k8s/kuttl-v2-ci-test.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
+startKIND: true
+kindContainers:
+  - localhost/redpanda-operator:dev
+  - docker.redpanda.com/redpandadata/redpanda:v23.2.11
+  - docker.redpanda.com/redpandadata/console:v2.3.2
+testDirs:
+  - ./tests/e2e-v2-ci
+kindConfig: ./kind-for-v2.yaml
+kindNodeCache: false
+commands:
+  - command: mkdir -p tests/_e2e_artifacts_v2
```

Comment:
NIT: If `src/go/k8s/kuttl-v2-test.yaml` would be executed after or before `src/go/k8s/kuttl-v2-ci-test.yaml` then this folder could be overwritten by the last kuttle execution.

The folder name needs to be different from 
https://github.com/redpanda-data/redpanda/blob/ec93e657f526be773985a51e2810081926cbf4e3/src/go/k8s/kuttl-v2-test.yaml#L11

I propose
```suggestion
  - command: mkdir -p tests/_e2e_helm_artifacts_v2
```

Reply:
So these are not tested in different docker contexts? Anyway yeah just in case we can do this.

Reply:
If I would run on my local then this would clash.

Reply:
done